testdata/m1_facilitator/facilitator_manifest.yml

```diff
@@ -0,0 +1,28 @@
+version: v0.14
+architecture_layout: M1
+personas:
+  - facilitator
+chain: 1405
+metachain:
+  origin:
+    avatar_account: "0xac7E36b3cdDb14Bf1c67dC21fFB24C73d03d8FF7"
+    node_endpoint: https://localhost.com:8545
+    graph_ws_endpoint: ws://localhost:8000/subgraphs/name/openst/ost-composer
+    graph_rpc_endpoint: http://localhost:8000/subgraphs/name/openst/ost-composer
```

Comment:
ost-composer?

Reply:
It's just a test endpoints. updated anyways.